### ISSUE TITLE
QUESTION: Why is the css of my datepicker plugin not loaded?

### ISSUE THREAD

**Issue by marvinstanley** [jeroennoten/Laravel-AdminLTE#374]
Hey, fairly new in using this implementation of AdminLTE and Laravel. Sorry if I misuse some terminologies.

Anyways, I've been trying to use the `datepicker` plugin to try and recreate the theme from the official [AdminLTE](https://adminlte.io/themes/AdminLTE/index.html) example.

I have configured the `config/adminlte.php` file to initialize my `datepicker` plugin like shown

```
        [
            'name' => 'Date Picker',
            'active' => true,
            'files' => [
                [
                    'type' => 'css',
                    'asset' => true,
                    'location' => 'vendor/adminlte/bower_components/bootstrap-datepicker/dist/css/bootstrap-datepicker.min.css',
                ],
                [
                    'type' => 'js',
                    'asset' => true,
                    'location' => 'vendor/adminlte/bower_components/bootstrap-datepicker/dist/js/bootstrap-datepicker.min.js',
                ],
            ],
        ],
```

And I have put this in the `@section('js')` at the blade file like shown 
```
@section('js')
<script>
    $(document).ready(function () {
      $("#calendar").datepicker();
    }); 
  </script>
@stop
```

Putting that in my blade file has at least made my `datepicker` plugin appeared, however the style of the `datepicker` plugin wasn't shown correctly, as if the `css` was not present.

The weirdest solution I currently found was to reference the location of the `datepicker` plugin again in the `@section('js')` like shown
```
@section('js')
  <script src="{{ asset('vendor/adminlte/bower_components/bootstrap-datepicker/dist/js/bootstrap-datepicker.min.js')}}"></script>

  <script>
    $(document).ready(function () {
      $("#calendar").datepicker();
    }); 
  </script>
@stop
```

While this works, as using the code snippet above has actually made the appearance of the `datepicker` plugin identical to the official [AdminLTE](https://adminlte.io/themes/AdminLTE/index.html) example, I believe there might be a more elegant solution than this.

What should I do to resolve this issue other than the current solution that I have provided.

Help is appreciated.
Stanley Marvin.

**Comment by andcarpi** [comment]
What's the AdminLTE and Laravel version?
Anyways, since the assets location are working when you inject then in your views js section, you are probably having issues in the plugin configuration. I coudn't spot the problem with the code you posted, so double check the plugins array openings/closings...

**Comment by marvinstanley** [comment]
Thanks for the reply @andcarpi .
I'm currently using `jeroennoten/Laravel-AdminLTE/ v1.27.2` and `Laravel 5.8`

Anyways, I've found the cause of the problem and the solution for it. 
The mistake I did to cause the problem was I accidentally initialized the `jQuery` plugin again in the `config/adminlte.php`..., which I obviously shouldn't have done as the `jQuery` plugin was already injected before the other plugins in the `master.blade.php` file.
The obvious solution was to simply remove the plugin initialization of the `jQuery` plugin in the `config/adminlte.php`...

Should have triple checked the plugins configuration..
Thank you again for replying.

Regards,
Stanley Marvin.
